Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM php:7-cli
+
+ARG COMPOSER_FLAGS="--prefer-dist --no-interaction"
+ARG DEBIAN_FRONTEND=noninteractive
+ENV COMPOSER_ALLOW_SUPERUSER 1
+ENV COMPOSER_PROCESS_TIMEOUT 3600
+
+WORKDIR /code
+
+COPY docker/php-prod.ini /usr/local/etc/php/php.ini
+COPY docker/composer-install.sh /tmp/composer-install.sh
+
+# Deps
+RUN apt-get update
+RUN apt-get install -y  --no-install-recommends \
+    wget \
+    curl \
+    make \
+    git \
+    bzip2 \
+    time \
+    libzip-dev \
+    zip \
+    unzip \
+    openssl \
+    vim
+
+# PHP
+RUN docker-php-ext-install sockets
```

Comment:
není tady vlastně potřeba ještě enable? https://github.com/docker-library/php/blob/master/docker-php-ext-enable

Reply:
Aha, to mozna nemusi. Skusim to vyhodit este